lib/req/steps.ex

```diff
@@ -71,6 +72,7 @@ defmodule Req.Steps do
       put_base_url: &Req.Steps.put_base_url/1,
       auth: &Req.Steps.auth/1,
       put_params: &Req.Steps.put_params/1,
+      put_path_params_style: &Req.Steps.put_path_params_style/1,
```

Comment:
could you merge this into the put_path_params step instead? It's going to be easier to document too, just another option for that step. Finally, I don't think we need to store it in req.private and instead look at req.options when applying params. I prefer to keep as least things as possible in req.private since it's user accessible. wdyt?

Reply:
Suresies, incoming

Reply:
@wojtekmach done!